### PR TITLE
Styled button hover and completed challenge 1

### PR DESCRIPTION
**Pull request by unclebay143**

### CODE REVIEW

File: home-page/style.css [hunk 1]
```diff
@@ -187,6 +187,11 @@ body::-webkit-scrollbar-thumb {
 .btn_div {
   text-align: center;
 }
+.btn:hover,
```
Comment (DavidMatthewFraser) [inline home-page/style.css:190]: Nice touch 👍

File: challenges/challenge-1/challenge-1.html [hunk 2]
```diff
@@ -87,6 +87,14 @@ <h1>Camper2</h1>
               Gandhi
             </p>
           </div>
+          <!-- cell for UncleBay -->
+          <div class="camperCell col-md-5 col-lg-3" id="unclebay">
```
Comment (DavidMatthewFraser) [inline challenges/challenge-1/challenge-1.html:91]: Looks great 👍

**Review by DavidMatthewFraser — APPROVED** [review]
Thanks for adding yourself to Contributors, doing challenge one, and adding button:hover to homepage buttons 👍
Welcome Aboard Ayodele 😄

File: home-page/style.css (hunk 1 above)
Comment (unclebay143) [reply home-page/style.css:190]: thanks

File: challenges/challenge-1/challenge-1.html (hunk 2 above)
Comment (unclebay143) [reply challenges/challenge-1/challenge-1.html:91]: glad you liked it

**Pull request merged**